code/game/objects/items/explosives/warhead.dm

```diff
@@ -9,10 +9,11 @@
 	name = "84mm rocket warhead"
 	desc = "A custom warhead meant for 84mm rocket shells."
 	icon_state = "warhead_rocket"
-	max_container_volume = 180
+	max_container_volume = 210
+	no_star_shape = TRUE
 	matter = list("metal" = 11250) //3 sheets
-	reaction_limits = list( "max_ex_power" = 300, "base_ex_falloff" = 120,"max_ex_shards" = 64,
-							"max_fire_rad" = 7, "max_fire_int" = 30, "max_fire_dur" = 36,
+	reaction_limits = list( "max_ex_power" = 215, "base_ex_falloff" = 90,"max_ex_shards" = 64,
+							"max_fire_rad" = 7, "max_fire_int" = 40, "max_fire_dur" = 48,
```

Comment:
```suggestion
							"max_fire_rad" = 6, "max_fire_int" = 40, "max_fire_dur" = 48,
```
Not really interested in the spec killing themselves even if they shoot at max range.